tsconfig.json

```diff
@@ -17,7 +17,9 @@
         /* :TCHAP: */
         "paths": {
             "~tchap-web/*": ["./*"],
-            "~tchap-web-dep/*": ["./yarn-linked-dependencies/*"]
+            "~tchap-web-dep/*": ["./yarn-linked-dependencies/*"],
```

Comment:
same here : I suggest `~matrix-react-sdk` and `~matrix-js-sdk`, it would be the simplest